actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -45,12 +45,22 @@ def action
       end
 
       def internal?
-        controller.to_s =~ %r{\Arails/(info|mailers|welcome)} || path =~ %r{\A#{Rails.application.config.assets.prefix}\z}
+        internal_controller? || internal_asset?
       end
 
       def engine?
         rack_app.respond_to?(:routes)
       end
+
+      private
+        def internal_controller?
+          controller.to_s =~ %r{\arails/(info|mailers|welcome)}
```

Comment:
I, personally prefer the double-slash syntax. I find it easier human readable, but I suppose that's a matter of internal rails style guide. 

And I have been keeping good, @bsodmike! I'm keeping an eye out for the new rails release. One of the more promising in a long time. Haven't been working much directly with rails for a while.